entrypoint.sh

```diff
@@ -48,11 +48,11 @@ if ! $(wp core is-installed --allow-root); then
rest_rates_35\";s:0:\"\";s:20:\"ar_interest_rates_36\";s:0:\"\";s:20:\"br_interest_rates_13\";s:0:\"\";s:20:\"br_interest_rates_14\";s:0:\"\";s:20:\"br_interest_rates_15\";s:0:\"\";s:20:\"br_interest_rates_16\";s:0:\"\";s:20:\"br_interest_rates_17\";s:0:\"\";s:20:\"br_interest_rates_18\";s:0:\"\";s:20:\"br_interest_rates_19\";s:0:\"\";s:20:\"br_interest_rates_20\";s:0:\"\";s:20:\"br_interest_rates_21\";s:0:\"\";s:20:\"br_interest_rates_22\";s:0:\"\";s:20:\"br_interest_rates_23\";s:0:\"\";s:20:\"br_interest_rates_24\";s:0:\"\";s:20:\"br_interest_rates_25\";s:0:\"\";s:20:\"br_interest_rates_26\";s:0:\"\";s:20:\"br_interest_rates_27\";s:0:\"\";s:20:\"br_interest_rates_28\";s:0:\"\";s:20:\"br_interest_rates_29\";s:0:\"\";s:20:\"br_interest_rates_30\";s:0:\"\";s:20:\"br_interest_rates_31\";s:0:\"\";s:20:\"br_interest_rates_32\";s:0:\"\";s:20:\"br_interest_rates_33\";s:0:\"\";s:20:\"br_interest_rates_34\";s:0:\"\";s:20:\"br_interest_rates_35\";s:0:\"\";s:20:\"br_interest_rates_36\";s:0:\"\";s:20:\"co_interest_rates_13\";s:0:\"\";s:20:\"co_interest_rates_14\";s:0:\"\";s:20:\"co_interest_rates_15\";s:0:\"\";s:20:\"co_interest_rates_16\";s:0:\"\";s:20:\"co_interest_rates_17\";s:0:\"\";s:20:\"co_interest_rates_18\";s:0:\"\";s:20:\"co_interest_rates_19\";s:0:\"\";s:20:\"co_interest_rates_20\";s:0:\"\";s:20:\"co_interest_rates_21\";s:0:\"\";s:20:\"co_interest_rates_22\";s:0:\"\";s:20:\"co_interest_rates_23\";s:0:\"\";s:20:\"co_interest_rates_24\";s:0:\"\";s:20:\"co_interest_rates_25\";s:0:\"\";s:20:\"co_interest_rates_26\";s:0:\"\";s:20:\"co_interest_rates_27\";s:0:\"\";s:20:\"co_interest_rates_28\";s:0:\"\";s:20:\"co_interest_rates_29\";s:0:\"\";s:20:\"co_interest_rates_30\";s:0:\"\";s:20:\"co_interest_rates_31\";s:0:\"\";s:20:\"co_interest_rates_32\";s:0:\"\";s:20:\"co_interest_rates_33\";s:0:\"\";s:20:\"co_interest_rates_34\";s:0:\"\";s:20:\"co_interest_rates_35\";s:0:\"\";s:20:\"co_interest_rates_36\";s:0:\"\";s:20:\"mx_interest_rates_13\";s:0:\"\";s:20:\"mx_interest_rates_14\";s:0:\"\";s:20:\"mx_interest_rates_15\";s:0:\"\";s:20:\"mx_interest_rates_16\";s:0:\"\";s:20:\"mx_interest_rates_17\";s:0:\"\";s:20:\"mx_interest_rates_18\";s:0:\"\";s:20:\"mx_interest_rates_19\";s:0:\"\";s:20:\"mx_interest_rates_20\";s:0:\"\";s:20:\"mx_interest_rates_21\";s:0:\"\";s:20:\"mx_interest_rates_22\";s:0:\"\";s:20:\"mx_interest_rates_23\";s:0:\"\";s:20:\"mx_interest_rates_24\";s:0:\"\";s:20:\"mx_interest_rates_25\";s:0:\"\";s:20:\"mx_interest_rates_26\";s:0:\"\";s:20:\"mx_interest_rates_27\";s:0:\"\";s:20:\"mx_interest_rates_28\";s:0:\"\";s:20:\"mx_interest_rates_29\";s:0:\"\";s:20:\"mx_interest_rates_30\";s:0:\"\";s:20:\"mx_interest_rates_31\";s:0:\"\";s:20:\"mx_interest_rates_32\";s:0:\"\";s:20:\"mx_interest_rates_33\";s:0:\"\";s:20:\"mx_interest_rates_34\";s:0:\"\";s:20:\"mx_interest_rates_35\";s:0:\"\";s:20:\"mx_interest_rates_36\";s:0:\"\";s:39:\"checkout_manager_colombia_document_type\";s:0:\"\";}" WHERE option_name="woocommerce_ebanx-global_settings"' --allow-root
```

Comment:
Why you change the option values?

Reply:
We had to update the docker manually because the seed is in the container and is not run in the start of the tests, we should probably review the whole test suite and docker situation as it's full of little problems like this.

Reply:
I changed the values because for some reason when I changed built the new container the ids of the pages changed.